README.md

```diff
@@ -19,12 +19,12 @@ A curated list of awesome packages and frameworks for implementing javascript ap
 
 ### Web
 
-* [Neutralino](https://github.com/neutralinojs/neutralinojs) - a runtime to build desktop apps using the system's own web browser (Windows, macOS, Linux)
-* [Electrino](https://github.com/pojala/electrino) - a runtime to build desktop apps using the system's own web browser (macOS only)
+* [Neutralino](https://github.com/neutralinojs/neutralinojs) - a runtime to build desktop apps using the system's own web browser (cross-platform)
+* [Electrino](https://github.com/pojala/electrino) - a runtime to build desktop apps using the system's own web browser (cross-platform)
```

Comment:
Does this support Linux?

Reply:
The documentation is nonexistent, i read through it a few times but i might be wrong

Reply:
So its not cross platform 😉